source/common/config/well_known_names.cc

```diff
@@ -211,6 +211,9 @@ TagNameValues::TagNameValues() {
 
   // http.<stat_prefix>.rbac.(<rules_stat_prefix>.)*
   addTokenized(RBAC_HTTP_PREFIX, "http.*.rbac.$.**");
+
+  // downstream_proxy_proto.versions.(<version>).**
+  addTokenized(PROXY_PROTOCOL_VERSION, "downstream_proxy_proto.versions.$.**");
```

Comment:
The prometheus stats come out much nicer with the following
```suggestion
  addRe2(PROXY_PROTOCOL_VERSION, R"(^downstream_proxy_proto\.(versions\.v(\d)\.)\w+)",
         "downstream_proxy_proto.versions");
```

Yields:

```
$ curl -s localhost:9901/stats/prometheus | grep proxy_p
# TYPE envoy_downstream_cx_proxy_proto_error counter
envoy_downstream_cx_proxy_proto_error{} 0
# TYPE envoy_downstream_proxy_proto_disallowed counter
envoy_downstream_proxy_proto_disallowed{envoy_proxy_protocol_version="1"} 0
envoy_downstream_proxy_proto_disallowed{envoy_proxy_protocol_version="2"} 0
# TYPE envoy_downstream_proxy_proto_error counter
envoy_downstream_proxy_proto_error{envoy_proxy_protocol_version="1"} 0
envoy_downstream_proxy_proto_error{envoy_proxy_protocol_version="2"} 0
# TYPE envoy_downstream_proxy_proto_found counter
envoy_downstream_proxy_proto_found{envoy_proxy_protocol_version="1"} 0
envoy_downstream_proxy_proto_found{envoy_proxy_protocol_version="2"} 0
# TYPE envoy_downstream_proxy_proto_not_found_allowed counter
envoy_downstream_proxy_proto_not_found_allowed{} 0
# TYPE envoy_downstream_proxy_proto_not_found_disallowed counter
envoy_downstream_proxy_proto_not_found_disallowed{} 0
```

Reply:
Good idea, thanks. I'll admit the syntax wasn't obvious, especially the nested capture groups. I'll file a separate PR to update the docs on this tomorrow.

Done.